app/webpacker/css/admin/products_v3.scss

```diff
@@ -36,12 +39,38 @@
 
     background-color: $color-tbl-bg;
     padding: 4px;
+    padding-top: 0; // Hide border because .form-actions is there. It is added with th padding instead.
     border-collapse: separate; // This is needed for the outer padding. Also should be helpful to give more flexibility of borders between rows.
 
     // Additional horizontal padding to align with input contents
-    thead th.with-input {
-      padding-left: $padding-tbl-cell + $hpadding-txt;
-      padding-right: $padding-tbl-cell + $hpadding-txt;
+    thead {
+      position: sticky;
+      top: 0;
+      z-index: 1; // TODO: Cover .popout and .vertical-ellipsis-menu, but only when sticky
```

Comment:
Shouldn't this use variable ?

Reply:
Yes. I was expecting to come back and do something more advanced here, but haven't so far. 
Hmm I will add it 👍

Reply:
Changed my mind: I reckon there's no need to register it when z-index is `1`. For anything higher, we should register a global variable.